driver-workload-executor/build.gradle

```diff
@@ -34,13 +40,21 @@ sourceSets {
 }
 
 dependencies {
-    api project(':driver-sync')
-
+    implementation project(':driver-sync')
+    implementation project(':driver-core').sourceSets.test.output
     implementation project(':driver-sync').sourceSets.test.output
-
     implementation 'ch.qos.logback:logback-classic:1.1.1'
+    implementation(platform("org.junit:junit-bom:5.6.2")) // TODO: grab BOM version from parent
```

Comment:
Is this todo still needed?

Reply:
Ah, I meant to comment to you about this.  Do you know how we can share the BOM version between here and the main build.gradle?

Reply:
Gotcha, check out how projectReactorVersion is set in the top level project and used in the reactive streams project.  That pattern should work here also.

Reply:
`junitBomVersion` needs to be added to the `ext` block and then it should be usable across the sub projects.

Reply:
Thanks, done.